pravega-cli/src/test/java/io/pravega/tools/pravegacli/ControllerDescribeStreamCommandTest.java

```diff
@@ -0,0 +1,232 @@
+        pravegaProperties.setProperty("cli.controllerGrpcUri", "localhost:" + CONTROLLER_PORT.toString());
+        pravegaProperties.setProperty("pravegaservice.zkURL", localPravega.getInProcPravegaCluster().getZkUrl());
+        pravegaProperties.setProperty("pravegaservice.containerCount", "4");
+        pravegaProperties.setProperty("cli.authEnabled", Boolean.toString(AUTH_ENABLED));
+        pravegaProperties.setProperty("cli.userName", "admin");
+        pravegaProperties.setProperty("cli.password", "1111_aaaa");
+        pravegaProperties.setProperty("cli.segmentStore.uri", Inet4Address.getLocalHost().getHostAddress().toString() + ":" + SEGMENT_STORE_PORT.toString());
+        STATE.get().getConfigBuilder().include(pravegaProperties);
+
+        localPravega.start();
+
+        // Wait for the server to complete start-up.
+        TimeUnit.SECONDS.sleep(20);
+    }
+
+    ClientConfig prepareValidClientConfig() {
+        return ClientConfig.builder()
+                .controllerURI(URI.create(localPravega.getInProcPravegaCluster().getControllerURI()))
+                .build();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        if (localPravega != null) {
+            localPravega.close();
+        }
+    }
+
+    @Test
+    public void testDescribeStreamCommand() throws Exception {
+        String scope = "testScope";
+        String testStream = "testStream";
+        ClientConfig clientConfig = prepareValidClientConfig();
+
+        // Generate the scope and stream required for testing.
+        @Cleanup
+        StreamManager streamManager = StreamManager.create(clientConfig);
+        assertNotNull(streamManager);
+
+        boolean isScopeCreated = streamManager.createScope(scope);
+
+        // Check if scope created successfully.
+        assertTrue("Failed to create scope", isScopeCreated);
+
+        boolean isStreamCreated = streamManager.createStream(scope, testStream, StreamConfiguration.builder()
+                .scalingPolicy(ScalingPolicy.fixed(1))
+                .build());
+
+        // Check if stream created successfully.
+        Assert.assertTrue("Failed to create the stream ", isStreamCreated);
+
+        String commandResult = executeCommand("controller describe-stream " + scope + " " + testStream, STATE.get());
+        Assert.assertTrue(commandResult.contains("stream_config"));
+        Assert.assertTrue(commandResult.contains("stream_state"));
+        Assert.assertTrue(commandResult.contains("segment_count"));
+        Assert.assertTrue(commandResult.contains("is_sealed"));
+        Assert.assertTrue(commandResult.contains("active_epoch"));
+        Assert.assertTrue(commandResult.contains("truncation_record"));
+        Assert.assertTrue(commandResult.contains("scaling_info"));
+    }
+
+    static String executeCommand(String inputCommand, AdminCommandState state) throws Exception {
```

Comment:
Why do you need this again? This is already in `TestUtils`, right?

Reply:
Need `executeCommand` to call the `execute` method from the `TestingDescribeStreamCommand` subclass in order to use the new overridden `instantiateSegmentHelper`. If `executeCommand` from `TestUtils` is called then the `execute` method from the original class will be used which is not suitable for testing against standalone.